guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,485 @@
+{
```

Comment:
<p>The saved model can be loaded back using <code>fcn.from_model(emd_path, data=None)</code> or by <code>MLModel.from_model(emd_path, data=None)</code>, which will load the trained model from the provided <code>emd_path</code> and would be ready for inferencing. The <code>data</code> parameter is optional <s>which</s> <u>and </u>requires a fastai Databunch returned  by the <code>prepare_tabulardata</code> function, <u>and is</u> only for further training the model on a new dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>